tezos/context/src/working_tree/working_tree.rs

```diff
@@ -11,29 +11,29 @@
 //! A storage with just one key `a/b/c` and its corresponding value `8` is represented like this:
 //!
 //! ``
-//! [commit] ---> [tree1] --a--> [tree2] --b--> [tree3] --c--> [blob_8]
+//! [commit] ---> [dir1] --a--> [dir2] --b--> [dir3] --c--> [blob_8]
```

Comment:
As mentioned in the above TODO, all this is quite out of date with the current implementation. It is not just about renaming, it needs to be adjusted (or replaced) so that it is still accurate.